File: core/src/main/java/cz/dynawest/jtexy/parsers/TexyParserModEvent.java

package cz.dynawest.jtexy.parsers;

import cz.dynawest.jtexy.modules.TexyModifier;

/**
 *
 * @author Ondrej Zizka
 */
public class TexyParserModEvent extends TexyParserEvent {

	protected TexyModifier modifier;
	public TexyModifier getModifier() {		return modifier;	}
	

	public TexyParserModEvent( TexyParser parser, String text, TexyModifier modifier ){
		super(parser, text);
		this.modifier = modifier;
	}



}

File: core/src/main/java/cz/dynawest/jtexy/parsers/TexyParserModEvent.kt
package cz.dynawest.jtexy.parsers

import cz.dynawest.jtexy.modules.TexyModifier

/**
 *
 * @author Ondrej Zizka
 */
open class TexyParserModEvent(parser: TexyParser?, text: String?, var modifier: TexyModifier?) : TexyParserEvent(parser, text)
